src/harness.js

```diff
@@ -393,7 +395,30 @@ class Harness {
   get consoleErrors() {
     return this._state.console
       .filter(msg => msg.type() !== 'log')
-      .filter(msg => msg.text() !== 'Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME');
+      .filter(msg => msg.text() !== 'Failed to load resource: net::ERR_UNKNOWN_URL_SCHEME')
+      .filter(msg => msg.type() !== 'warning');
```

Comment:
do you think this changes is still valuable? are people just trying to exclude that one warning or is there an actual broad need for this? are there other warnings that _should_ fail a test?